src/main/kotlin/com/hrothwell/anime/domain/UserSecrets.kt

```diff
@@ -5,5 +5,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class UserSecrets(
   val user_name: String,
-  val client_id: String
+  val client_id: String,
+//  val oauth_authorization_code: String?,
+//  val oauth_token: String? // TODO fine to keep this here?
```

Comment:
need to ignore missing keys or something using kotlinx, didn't wanna bother with it rn